### ISSUE TITLE
File moves in wpt repo are not reflected

### ISSUE THREAD

**Issue by nt1m** [web-platform-tests/wpt-metadata#2267]
See commits linked to https://github.com/web-platform-tests/wpt/issues/8615

They should be reflected here.

**Comment by foolip** [comment]
Renames are done as part of https://github.com/web-platform-tests/wpt-metadata/blob/master/.github/workflows/update_wpt_manifest.yml, and at least some renaming is happening in https://github.com/web-platform-tests/wpt-metadata/pull/2266

**Comment by foolip** [comment]
OK, looks like there's a bug. https://github.com/web-platform-tests/wpt-metadata/runs/4506562611?check_suite_focus=true is failing due to dir-style-03a.html not being in the manifest, and must not have detected the rename in https://github.com/web-platform-tests/wpt/pull/31997/files correctly.

**Comment by KyleJu** [comment]
with #2302, all metadata should be moved accordingly. I still have to look into why update_wpt_manifest.yml failed to rename the tests. Thanks for reporting the issue!

**Comment by foolip** [comment]
@KyleJu do you think there have been more deletions like https://github.com/web-platform-tests/wpt-metadata/pull/2260 in the past that we just haven't noticed yet?

**Comment by KyleJu** [comment]
> @KyleJu do you think there have been more deletions like #2260 in the past that we just haven't noticed yet?

Should't be. For #2260, I deleted them manually and thought deletion didn't work properly in the GitHub actions. I should have circled back and dug into it. In general I screen the change manually myself

**Comment by gsnedders** [comment]
This leaves oddities with Interop 2022; for example, see f9afd657dbe6e1b338fdd496d08897ad029cface which dealt with /css/css-scroll-snap/scroll-snap-stop.html being renamed to /css/css-scroll-snap/scroll-snap-stop-001.html: this now means that earlier Interop 2022 runs (i.e., where the wpt.fyi data predates the rename) don't include the test, despite it having existed since 2019.

**Comment by foolip** [comment]
Yes, we keep metadata in sync with WPT and the Interop 2022 scoring code doesn't follow renames in the past, so a renamed test will behave exactly like a new test.

I don't have any (good) ideas for how to improve on this.
